pkg/objectstore/migrate_test.go

```diff
@@ -91,7 +91,9 @@ func Test_updateKotsadmObjectStore(t *testing.T) {
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
 			req := require.New(t)
-			logs := nilLogger()
+			logs := func(fmt string, args ...interface{}) {
```

Comment:
once again why do you need to create a function here? cant you just pass in t.Logf?